turbo.json

```diff
@@ -5,11 +5,15 @@
         "cache": false,
         "dependsOn": ["build"]
       },
+      "test:system": {
```

Comment:
Is it worth adding the system test directory as an input?

Reply:
By default, Turbo sets the inputs to anything that's included in source control that isn't gitignored, so it should already be included as part of the cache hashing algorithm. - https://turbo.build/repo/docs/core-concepts/caching/file-inputs#customizing-the-behavior